src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -78,6 +129,24 @@ def _assess_clusters(self, all_clusters):
                 for value in cluster.spark_conf.values():
                     if "dbfs:/mnt" in value or "/dbfs/mnt" in value:
                         failures.append(f"using DBFS mount in configuration: {value}")
+
+                # Checking if Azure cluster config is present in spark config
+                for conf in _AZURE_SP_CLUSTER_CONF:
+                    if re.search(conf, str(cluster.spark_conf)):
```

Comment:
Can you compare it key-by-key, instead of naively comparing the string dump of the conf, with may have hidden bugs